src/Fixer/Operator/TernaryToNullCoalescingFixer.php

```diff
@@ -89,6 +89,10 @@ private function fixIsset(Tokens $tokens, int $index): void
             return; // some weird stuff inside the isset
         }
 
+        if ($issetTokens->generateCode() === '$this') {
+            return; // null coalescing operator does not with $this
```

Comment:
Please, could we add new tests to check introduced changes and ensure it won't be broken in the future (due to, for example, to some refactoring) ?

Reply:
Added.